mentoring/dataexport.py

```diff
@@ -23,25 +23,42 @@
 
 # Imports ###########################################################
 
+import json
 import logging
+import unicodecsv
 
-from itertools import groupby
+from StringIO import StringIO
 from webob import Response
 from xblock.core import XBlock
 from xblock.fields import String, Scope
 from xblock.fragment import Fragment
+from xblockutils.resources import ResourceLoader
 
-from .models import Answer
-from .utils import list2csv, loader
-
+from .components import AnswerBlock
 
 # Globals ###########################################################
 
 log = logging.getLogger(__name__)
+loader = ResourceLoader(__name__)
+
+# Utils ###########################################################
+
+
+def list2csv(row):
+    """
+    Convert a list to a CSV string (single row)
+    """
+    f = StringIO()
```

Comment:
@e-kolpakov I tried a context manager but it turns out that `StringIO` doesn't support that.